docs/console/usage-billing.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: usage-billing
+title: Usage-based billing
+sidebar_label: Usage-based billing
+slug: usage-billing
+---
+
+# Usage-based billing
+
+:::info
+
+This feature is in preview. While we are rolling out usage-based billing, we will not collect payment for overusage. Payment
+collection will start XX.XX.2024.
+
+Please report any issues to support@ory.sh or contact us in the [Ory Community](https://slack.ory.sh/).
+
+:::
+
+## Overview
+
+Usage-based invoices is a new feature in the Ory Console that you to understand your usage of Ory Cloud services, the usage that
+is included in your plan, and the overuse. Usage-based invoices are generated on a monthly basis.
+
+## Enabling Usage-based Invoices
+
+Displaying the usage-based invoices is guarded by a feature toggle and is disabled by default. To enable it, navigate to the
+[preview features page](https://console.ory.sh/projects/current/features) and enable the `Show invoices` feature. The Ory Console
+should reload itself and you should now see the usage-based invoices sidebar entry in the workspace settings page.
+
+### Ory Console UI
+
+After enabling the feature toggle, navigate to the workspace usage and billing page either through the workspace menu on the top
+left, or by navigating to the workspace settings page and then selecting `Usage and billing` from the sidebar.
+
+Usage invoices are only available for workspaces with a subscription. The usage will always be invoiced monthly, even if the plan
```

Comment:
```suggestion
Usage invoices are only available for workspaces that have an active subscription plan. The usage will always be invoiced monthly, even if the plan
```

Reply:
Technically, the subscription does not need to be active, so that you can also view older invoices. I'll reword this.